ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/validation/CustomValidators.java

```diff
@@ -100,8 +98,7 @@
   SHOW_COLUMNS(ShowColumns.class, ListSourceExecutor::columns),
   EXPLAIN(Explain.class, ExplainExecutor::execute),
   DESCRIBE_FUNCTION(DescribeFunction.class, DescribeFunctionExecutor::execute),
-  DESCRIBE_CONNECTOR(DescribeConnector.class,
-      new DescribeConnectorExecutor(new DefaultConnectServerErrors())::execute),
+  DESCRIBE_CONNECTOR(DescribeConnector.class, StatementValidator.NO_VALIDATION),
```

Comment:
I've removed this validation because it was a no-op -- validation only fails if an error is thrown, and no error was ever thrown from `DescribeConnectorExecutor::validate` since it returned a 200 response with an error entity on error, instead of throwing.

It doesn't make sense to validate this request because there's nothing to validate (validating it amounts to issuing the request to Connect, which is what execution does). The same logic applies to DropConnector and ListConnectors, both of which are already not being validated.

See https://github.com/confluentinc/ksql/pull/8999 for more on validating connector statements.

Reply:
Good catch, ``DescribeConnectorExecutor::execute`` never throws an exception indeed.

I guess we should do the same for LIST_VARIABLES and DESCRIBE_FUNCTION -- can do it in a follow-up